app/utils.py

```diff
@@ -1,11 +1,16 @@
 from flask import url_for as base_url_for
 from flask import abort, request
 from six import iteritems, string_types
+import os
 from werkzeug.exceptions import BadRequest
 
 from .validation import validate_updater_json_or_400
 
 
+def random_positive_public_id():
+    return int.from_bytes(os.urandom(8), 'big') >> 1
```

Comment:
You can just use `random.SystemRandom` y'know...

Reply:
/shrug